javascript/builtins/Temporal/Now.json

```diff
@@ -261,7 +261,7 @@
               }
             }
           },
-          "timeZoneID": {
+          "timeZoneId": {
```

Comment:
Let's move this to a separate PR, as it's a rename.

Reply:
done in https://github.com/mdn/browser-compat-data/pull/25714/commits/8258a6f6b0cba93abc3b48914eee57b4f217d715